src/modules/land_detector/LandDetector.h

```diff
@@ -127,10 +127,10 @@ class LandDetector
 	static constexpr uint32_t LAND_DETECTOR_UPDATE_RATE_HZ = 50;
 
 	/* Time in us that landing conditions have to hold before triggering a land. */
-	static constexpr uint64_t LAND_DETECTOR_TRIGGER_TIME_US = 2000000;
+	static constexpr uint64_t LAND_DETECTOR_TRIGGER_TIME_US = 2e6;
```

Comment:
if you cast it to an integer explicitly the compiler does never output a double in the assembly